src/main/java/org/folio/innreach/domain/service/impl/ItemServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.innreach.domain.service.impl;
+
+import static org.folio.innreach.util.ListUtils.getFirstItem;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import org.folio.innreach.client.InventoryClient;
+import org.folio.innreach.domain.dto.folio.inventory.InventoryItemDTO;
+import org.folio.innreach.domain.service.ItemService;
+
+@Service
+@RequiredArgsConstructor
+public class ItemServiceImpl implements ItemService {
+
+  private final InventoryClient inventoryClient;
+
+  @Override
+  public InventoryItemDTO create(InventoryItemDTO item) {
+    return inventoryClient.createItem(item);
+  }
+
+  @Override
+  public InventoryItemDTO update(InventoryItemDTO item) {
+    inventoryClient.updateItem(item.getId(), item);
+    return item;
+  }
```

Comment:
Is there any limitations why we can't receive and return the response of `inventoryClient.updateItem`?

Reply:
Yes there is. Whoever defined those methods should have checked the API documentation. According to it, PUT method returns no body but only Http 204 status
https://s3.amazonaws.com/foliodocs/api/mod-inventory/p/inventory.html#inventory_items__itemid__put

![image](https://user-images.githubusercontent.com/42244720/145430471-ff539bc7-fe03-4e4b-9542-15fa19963368.png)

Reply:
Yep, you are right)